tiatoolbox/utils/misc.py

```diff
@@ -75,3 +76,22 @@ def save_yaml(input_dict, output_path="output.yaml"):
     """
     with open(pathlib.Path(output_path), "w") as yaml_file:
         yaml.dump(input_dict, yaml_file)
+
+
+def cv2_imwrite(image_path, cv_im):
```

Comment:
Just wondering if it's better to just call it 'imwrite' rather than 'cv2_imwrite'? So that if later we find a better / faster way to write to file etc we can change the code here and the name would still work e.g. if it wasn't using cv2 anymore. Happy if you want to leave it as is though, just thinking about abstracting away exactly which library is doing the writing.

Reply:
I agree I think we should go with `imwrite`. Revised it in this commit https://github.com/TIA-Lab/tiatoolbox/commit/16ad9f93aa21b23344cb6261ff9f037e6d3646ab